Translation corrections (#413)
07th-mod/higurashi-console-arcs: Update/ura_001.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>反町</color>", NULL, "<color=#f5e6d3>Sorimachi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_sorima1009", 256, TRUE);
	OutputLine(NULL, "「まぁな…帰りにちょっと立ち飲み屋で、軽く引っかける程度か」",

> 			NULL, "\"You could say that... there's a small standing bar on the way home that I stop at once in a while, though.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"You could say that... There's a small standing bar on the way home that I stop at once in a while, though.\"", Line_Normal);
```

//rもっとも、それこそが二人の仲睦まじさの裏返しであり、多少嫉妬深いところも深い愛情があったからだろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "もっとも、それこそが二人の仲睦まじさの裏返しであり、多少嫉妬深いところも深い愛情があったからだろう。",

> 			NULL, "However, that was the flip side of their close relationship, and she probably got jealous exactly because of how much she loved him.", Line_Normal);

Reviewer: ```suggestion
			NULL, "However, that was the flip side of their close relationship, and she probably got jealous precisely because of how much she loved him.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>反町</color>", NULL, "<color=#f5e6d3>Sorimachi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_sorima1056", 256, TRUE);
	OutputLine(NULL, "「この程度のことでご機嫌をとらなきゃ、なんていちいち気を回してたら、財布にいくら金が詰まってても足りないぜ？",

> 			NULL, "\"If you had to cheer me up every time I got torn up over something like this, no amount of money in your wallet would be enough, would it now?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"If you had to cheer me up every time I got torn up over something like this, no amount of money in your wallet would be enough.", Line_WaitForInput);
```

			NULL, "\"As the special investigation department at the District Prosecutors Office, crushing that organization is our responsibility.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_sorima1076", 256, TRUE);
	OutputLine(NULL, "正義云々を語る以前に、法治国家の番人として…な」",

> 			NULL, " As the guardians of the law we are before speakers of justice... right?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Even before we are speakers of justice, we are guardians of the rule of law... right?\"", Line_Normal);
```

//r無意識のうちに剣呑な口調になってしまったのも、きっと私には言えない疲れや、悩みを抱えているからなのかもしれない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "無意識のうちに剣呑な口調になってしまったのも、きっと私には言えない疲れや、悩みを抱えているからなのかもしれない。",

> 			NULL, "I'm sure the reason she slipped into a nasty tone is because she's got things weighing on her mind that she can't tell me.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I'm sure the reason she slipped into a nasty tone is because she has things weighing on her mind that she can't tell me.", Line_Normal);
```
